diff -urN aa/conglomerate-0.7.15/doc/reference/Makefile.am conglomerate-0.7.15/doc/reference/Makefile.am
--- aa/conglomerate-0.7.15/doc/reference/Makefile.am	2003-12-22 22:16:14.000000000 +0100
+++ conglomerate-0.7.15/doc/reference/Makefile.am	2004-11-01 17:39:21.105941464 +0100
@@ -45,7 +45,8 @@
 # CFLAGS and LDFLAGS for compiling scan program. Only needed if your app/lib
 # contains GtkObjects/GObjects and you want to document signals and properties.
 GTKDOC_CFLAGS = $(CONGLOMERATE_CFLAGS) -I$(top_srcdir)/src
-GTKDOC_LIBS = $(CONGLOMERATE_LIBS) $(GTKSOURCEVIEW_LIBS) $(PRINTING_LIBS) \
+GTKDOC_LIBS = $(CONGLOMERATE_LIBS) \
+	$(GTKSOURCEVIEW_LIBS) $(ENCHANT_LIBS) $(PRINTING_LIBS) \
 	$(filter-out ../../src/main.o,$(wildcard ../../src/*.o))
 
 GTKDOC_CC=$(LIBTOOL) --mode=compile $(CC)
