diff -Nuard conglomerate-0.7.6.orig/data/conglomerate.desktop.in conglomerate-0.7.6/data/conglomerate.desktop.in
--- conglomerate-0.7.6.orig/data/conglomerate.desktop.in	2003-06-26 00:25:30.000000000 +0200
+++ conglomerate-0.7.6/data/conglomerate.desktop.in	2003-11-28 20:35:42.479789248 +0100
@@ -5,4 +5,5 @@
 Icon=conglomerate-icon-16.png
-Terminal=0
+Terminal=false
 Type=Application
-Categories=Application;Office;TextEditor;
+Categories=Application;Utility;TextEditor;
+Encoding=UTF-8
\ Brak znaku nowej linii na koñcu pliku
